static/src/javascripts/projects/common/modules/spacefinder.ts

```diff
@@ -116,19 +113,9 @@ const onImagesLoaded = memoize((rules: SpacefinderRules) => {
 				img.addEventListener('load', resolve);
 			}),
 	);
-	return Promise.all(imgPromises);
+	return Promise.all(imgPromises).then(() => Promise.resolve());
```

Comment:
If you turn this into an `async` function, you could then simply await this call. I think this would be more explicit.

```suggestion
	await Promise.all(imgPromises);
```

Reply:
This is a good suggestion, but this improvement seem separate from the the work here, so in the interest of keeping this on point, I am going to respectfully decline to take this action.